.travis.yml

```diff
@@ -1,14 +1,16 @@
 language: php
 php:
   - 5.6
+  - 7.0
 
 services:
   - redis-server
 
 before_script:
   - echo "apc.enabled = 1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
   - echo "apc.enable_cli = 1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
-  - echo "extension = apcu.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
+  - if [[ $TRAVIS_PHP_VERSION = 5.* ]]; then echo yes | pecl install -f apcu-4.0.11; fi
```

Comment:
Any particular reason to lock on a specific apcu version? `pecl` should be able to detect the latest version compatible with the `PHP` version

Reply:
version 5+ has the `APCUIterator` and that led to some weird errors, so i just based this on the Symfony Polyfill lib, I leave it to @bracki to decide how he wants to treat this since he had made the earlier choice to use apcu in 5.6 which is already unusual.